plugins/quota/validate.go

```diff
@@ -43,6 +70,26 @@ func isQuotaEnabled(v *volume.Volinfo) bool {
 	return false
 }
 
+// isQuotadRunning returns true for running and false for
+// failures and not running on that machine.
+// It is not volume based.
+func isQuotadRunning() (bool, error) {
```

Comment:
This can be a function that only returns true or false.

Reply:
the err is necessary to handle the following failure.

 77         quotadDaemon, err := NewQuotad()
 78         if err != nil {
 79                 return false, err
 80         }

Reply:
That error is possible only if quota (glusterfs) isn't installed on the system. You can ignore that. If quota isn't installed, the validate in rest handler will return "option not found" error.

Reply:
if so then i'll remove it and send a single return value and handle is as per that.

Reply:
and if thats the case can i remove it the err check for NewQuotad here?